app/controllers/api/v1/admin/base_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Admin::BaseController < Api::V1::ApiController
+
```

Comment:
Could we put the `before_filter` here so all children controllers run it by default? That way if we do have an endpoint that is different, we have to explicitly skip it?

In other words, put ` before_action :require_admin` here and remove it from the child controllers. WDYT?